keadm/cmd/keadm/app/cmd/helm/cloudcore.go

```diff
@@ -0,0 +1,325 @@
+/*
+Copyright 2019 The KubeEdge Authors.
```

Comment:
2023 or 24?

Reply:
I added the license header to all new files and changed the date to 2024.